### DISCUSSION TITLE
Questions about Self Hosting

### DISCUSSION THREAD

**Discussion by jamesliu08** [deepgram/community#875] [General help]
My team is looking into a self-hosted deployment of Deepgram. I see the minimum recommended specs for a node are listed [here](https://developers.deepgram.com/docs/self-hosted-deployment-environments) but I don't see any guidance on what kind of load a single node can handle. I'm wondering if Deepgram has any general numbers they could share for how many streams a single node could be expected to handle simultaneously? Our traffic is almost entirely streaming speech-to-text.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.
